### ISSUE TITLE
About epoch setting.

### ISSUE THREAD

**Issue by tiancai-call** [farewellthree/STAN#4]
Thanks for your great work！
I didn't find any statement about the number of epochs in the paper. In fact, given the training strategy used, providing only the learning rate does not seem to be of much use.

**Comment by farewellthree** [comment]
We set 20 epochs on retrieval and 50 epochs on recognition.
